### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by gosro** [freelensapp/freelens#1680]
### Describe the bug

Failed to refresh accessibility: RangeError: Maximum call stack size exceeded

### Freelens version

1.8.1
